### ISSUE TITLE
Incorrect documentation

### ISSUE THREAD

**Issue by pvegh** [Kotlin/kotlinx-kover#32]
Following https://github.com/Kotlin/kotlinx-kover#plugin-configuration
leads to an error:
> Could not find method test() for arguments [build_xxx$_run_closure4@70489108] on task set of type org.gradle.api.internal.tasks.DefaultTaskContainer.

**Comment by ptsiogas** [comment]
@pvegh I managed to make it work with this:

```
tasks.withType(Test) {
    kover {
        generateXml = false
        generateHtml = true
        coverageEngine = 'INTELLIJ'
        xmlReportFile.set(file("$buildDir/custom/report.xml"))
        htmlReportDir.set(file("$buildDir/custom/html"))
        binaryReportFile.set(file("$buildDir/custom/result.bin"))
    }
}
```

**Comment by qwwdfsad** [comment]
Right, it has a typo for Groovy, it has to be `tasks.withType(Test) `.

It will be reworked and fixed by #33
